airdrop/processor/loyalty_airdrop.py

```diff
@@ -21,14 +21,16 @@ def __init__(self, airdrop_id, airdrop_window_id=None):
     def format_user_registration_signature_message(self, address, signature_parameters):
         block_number = signature_parameters["block_number"]
         cardano_address = signature_parameters["cardano_address"]
+        cardano_wallet_name = signature_parameters["cardano_wallet_name"]
         formatted_message = USER_REGISTRATION_SIGNATURE_LOYALTY_AIRDROP_FORMAT
         formatted_message["message"] = {
             "Airdrop": {
                 "airdropId": self.airdrop_id,
                 "airdropWindowId": self.airdrop_window_id,
                 "blockNumber": block_number,
                 "walletAddress": address,
-                "cardanoAddress": cardano_address
+                "cardanoAddress": cardano_address,
+                "cardano_wallet_name": cardano_wallet_name
```

Comment:
can we name the key consistent i.e camelcase here

Reply:
Done